src/models/moderations.ts

```diff
@@ -132,6 +132,23 @@ const moderations = (sequelize: Sequelize) => {
         return result.map((r: any) => r.toJSON() as ModerationJSON);
     };
 
+    const findAllLikesByReference = async (
+        reference: string,
+        offset = 0,
+        limit = 20,
+        order: 'DESC' | 'ASC' = 'DESC'
+    ): Promise<string[]> => {
+        const result = await model.findAll({
+            attributes: ['creator'],
+            where: { reference, subtype: 'LIKE' },
+            offset,
+            limit,
+            order: [['createdAt', order]],
+        });
+
+        return result.map((r: any) => r.toJSON().creator);
```

Comment:
let's return `messageId` instead of `creator`

Reply:
then it is up to the client to extract the creator from the messageId `creator/hash`?

Reply:
For the likes and followers it would work, but for followings?
https://github.com/zkitter/zkitterd/pull/21#discussion_r1000470190

Reply:
btw let me merge the follower/followings stuff in that branch